docs/Create_Frontend_Unit_Tests.md

```diff
@@ -0,0 +1,135 @@
+[**Table of Contents**](index.md)
+
+![unit test](md_Graphics/testing_sm.jpg)
+
+# Create **Frontend** Unit Tests
```

Comment:
One misnomer I am seeing a bit here is mixing a technology with a specific part of the stack. Jest is used to test *JS* code primarily and also is being used as a runner for Fable / F# code.

Reply:
I removed the word: "Frontend"